Dockerfile

```diff
@@ -1,62 +1,52 @@
-# The official python:2.7 image no longer receives automatic rebuilds (it's
-# a year old as of October, 2021), so use the latest LTS release of Ubuntu
-# that includes Python 2.7 instead.
-FROM ubuntu:20.04
-
-# Install packages needed to run your application (not build deps).
-RUN set -x \
-    && RUN_DEPS=" \
-    ca-certificates \
-    git \
-    libpq5 \
-    make \
-    postgresql-client \
-    python2.7 \
-    " \
-    && apt-get update && apt-get install -y --no-install-recommends $RUN_DEPS \
+# pull official base image
+FROM python:3.8-slim-bullseye
+
+# set work directory
+WORKDIR /code
+
+# set environment varibles
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+# install deb packages
+RUN apt-get update \
+    && apt-get install --assume-yes --no-install-recommends \
+        git \
+        make \
+        postgresql-client-13 \
```

Comment:
Do we need to pin to `postgresql-client-13`?

Reply:
Probably not, this was me copy/pasting from the docker image of djangoproject.com. I'll try the change and see if the image still builds (and runs).